readme.md

```diff
@@ -186,6 +187,8 @@
 - [ShellGPT](https://github.com/TheR1D/shell_gpt) - Use ChatGPT from the command-line.
 - [chatGPT-shell-cli](https://github.com/0xacx/chatGPT-shell-cli) - Use ChatGPT from the command-line. Shell script.
 - [aifiles](https://github.com/jjuliano/aifiles) - Organize and manage your files using AI.
+- [lobe-commit](https://github.com/lobehub/lobe-commit/tree/master/packages/lobe-commit) - CLI GUI tool, uses ChatGPT to generate Gitmoji-based commit messages.
+- [lobe-i18n](https://github.com/lobehub/lobe-commit/tree/master/packages/lobe-i18n) - CLI GUI tool, automates the i18n translation process.
```

Comment:
"CLI GUI" is moot as it's already under the "CLI tools" category.

Reply:
www.meunumerovirtual.com